packages/api-core/src/site-scan/mutators.ts

```diff
@@ -1,6 +1,16 @@
 import { wpcom } from '../wpcom-fetcher';
 import type { SiteScan, ThreatActionOptions } from './types';
 
+export async function bulkFixThreats( siteId: number, threatIds: number[] ): Promise< SiteScan > {
```

Comment:
Could we name it `fixThreats` instead? We can add it after `fixThreat()` as they are related.

Reply:
We could, but should we? The purpose of the `bulk` prefix was to avoid exactly this situation: having only a single-character difference between `fixThreat()` and `fixThreats()`.

Reply:
I've updated it.